src/api/migrations/0008_auto_20200903_2034.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.12 on 2020-09-03 20:34
+
+import django.core.validators
+from django.db import migrations, models
+from ..models.view_models import model_view_migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0007_auto_20200819_2130'),
+    ]
+
+    operations = [
+        migrations.RunSQL(model_view_migrations.reverse_sql(), model_view_migrations.forward_sql()),
+        migrations.AlterField(
+            model_name='benthictransect',
+            name='len_surveyed',
+            field=models.DecimalField(decimal_places=1, max_digits=4, validators=[django.core.validators.MinValueValidator(10), django.core.validators.MaxValueValidator(100)], verbose_name='transect length surveyed (m)'),
```

Comment:
I think we shouldn't put model-level validators on this; submission validations should take care of warnings